GVFS/GVFS.Platform.Windows/BitLockerHelpers.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Management;
+
+namespace GVFS.Platform.Windows
+{
+    internal static class BitLockerHelpers
+    {
+        public const string VolumeEncryptionNamespace = @"root\cimv2\security\MicrosoftVolumeEncryption";
+
+        public static bool TryGetVolumeLockStatus(string driveName, out bool isLocked)
+        {
+            string queryString = $"SELECT * FROM Win32_EncryptableVolume WHERE DriveLetter = '{driveName}'";
+
+            using (var searcher = new ManagementObjectSearcher(VolumeEncryptionNamespace, queryString))
+            {
+                ManagementObjectCollection results = searcher.Get();
+                ManagementObject mo = results.OfType<ManagementObject>().FirstOrDefault();

```

Comment:
Please avoid abbreviated variable names

Reply:
Sure. I usually don't use abbreviated variable names, but based on [this](https://github.com/mjcheetham/VFSForGit/blob/723f21ff814a0003feae614a82763d8c2800951a/GVFS/GVFS.Platform.Windows/WindowsPhysicalDiskInfo.cs#L100) prior art I thought it was acceptable for `ManagementBaseObject` and `ManagementObject` to be `mbo`/`mo`.

Will change to `win32CryptoVolume` or similar. Should the other instances of `mbo` short var names (such as in WindowPhysicalDiskInfo.cs) be expanded too?

Reply:
I'm fairly certain I asked for no abbreviations on that other code as well :-). I do like your suggested rename, as it describe the thing the object represents, rather than its type in the API.